harmonica/forward/point_mass.py

```diff
@@ -0,0 +1,292 @@
+"""
+Forward modelling for point masses
+"""
+import numpy as np
+from numba import jit
+
+from ..constants import GRAVITATIONAL_CONST
+
+
+def point_mass_gravity(coordinates, point_mass, mass, field, dtype="float64"):
+    """
+    Compute gravitational fields of a point mass on spherical coordinates.
+
+    Parameters
+    ----------
+    coordinates : list or array
+        List or array containing `longitude`, `latitude` and `radius` of computation
+        points defined on a spherical geocentric coordinate system.
+        Both `longitude` and `latitude` should be in degrees and `radius` in meters.
+    point_mass : list or array
+        Coordinates of the point mass: [`longitude`, `latitude`, `radius`] defined on
+        a spherical geocentric coordinate system.
+        Both `longitude` and `latitude` should be in degrees and `radius` in meters.
+    mass : float
+        Mass of the point mass in kg.
+    field : str
+        Gravitational field that wants to be computed.
+        The available fields are:
+
+        - Gravitational potential: ``potential``
+        - Accelerations or gradient components: ``gx``, ``gy``, ``gz``
+        - Maurssi tensor components: ``gxx``, ``gxy``, ``gxz``, ``gyy``, ``gyz``,
+          ``gzz``
```

Comment:
It's best to put `(gravity gradient)` since not everyone uses the Marussi name.

Reply:
We should also move away from `x, y, z` in the names. Maybe change to `n, e, r` in this case. That's more straight forward.

Reply:
See the comment bellow.